demos/interactive_face_detection_demo/detectors.cpp

```diff
@@ -152,19 +153,36 @@ CNNNetwork FaceDetection::read(const InferenceEngine::Core& ie)  {
     }
     DataPtr& _output = outputInfo.begin()->second;
     output = outputInfo.begin()->first;
+    size_t num_classes = 0ul;
+
+    if (auto ngraphFunction = network.getFunction()) {
+        for (const auto op : ngraphFunction->get_ops()) {
```

Comment:
Would it be possible to only iterate over output ops, so that we don't have to go over the entire graph?

Reply:
Ping?

Reply:
I answered - no!

Reply:
Well, why not?

Reply:
If you want to play with it - do it in a separate MR. My MR is correct and I don't want to answer such questions anymore.

Reply:
BTW, `output ops` in ngraph are different from classical ones. I mean in ngraph we have dedicated Result op which denotes output layer, while classical CNNLayer based CNNNetwork does not have such ones. ngraph allows to iterate over results while we need nodes which produce this result and iterating over them is not convinient

Reply:
OK.